config/locales/en.yml

```diff
@@ -466,6 +468,14 @@ en:
         confirm: Are you sure you want to make your form live?
       group_member_input:
         member_email_address: Enter the email address of the person you want to add
+        role: Select the role for this person
```

Comment:
The content here doesn't match the content in the Figma - which one is correct?

Reply:
This is what I'm looking at in the Figma, for reference

<img width="628" alt="Screenshot 2024-05-14 at 15 57 11" src="https://github.com/alphagov/forms-admin/assets/5648592/4de5eabf-6238-4f2f-85b2-b9737f1226f8">

Reply:
totally right! the figma was correct and I've updated it